bin/varnishtest/vtc_http2.c

```diff
@@ -2397,6 +2412,48 @@ cmd_rxpush(CMD_ARGS)
 	} while (rcv < times || (loop && !(f->flags & END_HEADERS)));
 	s->frame = f;
 }
+ 
+/* SECTION: stream.spec.winup_rxwinup rxwinup
+ *
+ * Receive a WINDOW_UPDATE frame.
+ */
+static void
+cmd_rxwinup(CMD_ARGS)
+{
+	struct stream *s;
+	struct frame *f;
+	CAST_OBJ_NOTNULL(s, priv, STREAM_MAGIC);
+	s->frame = rxstuff(s);
+	CAST_OBJ_NOTNULL(f, s->frame, FRAME_MAGIC);
+	CHKFRAME(f->type, TYPE_WINDOW_UPDATE, 0, *av);
+	if (s->id == 0)
+		s->hp->h2_win_peer->size += s->frame->md.winup_size;
+	s->win_peer += s->frame->md.winup_size;
+}
+
+/* SECTION: stream.spec.settings_rxsettings rxsettings
+ *
+ * Receive a SETTINGS frame.
+ */
+static void
+cmd_rxsettings(CMD_ARGS)
+{
+	struct stream *s, *_s;
```

Comment:
I understand you are just following the code style in this area, but we should avoid naming variables `_s`.

Reply:
Let's not do it in this pull request.